active/0000-addon-options.md

```diff
@@ -0,0 +1,33 @@
+- Start Date: 2015-03-24
+- RFC PR: 
+- ember-cli Issue: 
+
+# Summary
+
+Doing something similar to what Yeoman does would be awesome.
```

Comment:
please actually elaborate, these RFC's are meant to flesh out the details so implementation is clear.